gapii/cc/spy.cpp

```diff
@@ -460,6 +487,24 @@ void Spy::gvr_frame_submit(CallObserver* observer, gvr_frame** frame,
   GvrSpy::gvr_frame_submit(observer, frame, list, head_space_from_start_space);
 }
 
+bool Spy::checkEndTrace() {
```

Comment:
I am not sure if "check" is the right word for this, this does not really check anything it is really more "EndTraceIfNeeded" or something like that. I would expect this to return true, if the trace was supposed to end (or had ended maybe), and true otherwise, but not both.

Reply:
Good point, I was unsure about that function name. Renamed.